packages/cw4/src/hook.rs

```diff
@@ -10,17 +10,13 @@ use cosmwasm_std::{to_binary, Binary, CosmosMsg, HumanAddr, StdResult, WasmMsg};
 /// old = Some, new = None -> Delete
 #[derive(Serialize, Deserialize, Clone, PartialEq, JsonSchema, Debug)]
 pub struct MemberDiff {
-    pub key: HumanAddr,
+    pub key: Addr,
```

Comment:
Wait, this is a message, not state.
Again, use String.
`Addr` only in state and env/info from the runtime (already trusted)